ci_watson/artifactory_helpers.py

```diff
@@ -515,21 +517,20 @@ def generate_upload_params(results_root, updated_outputs, verbose=True):
 
     # Create instructions for uploading results to artifactory for use
     # as new comparison/truth files
-    time_now = datetime.datetime.now()
     testname = os.path.split(os.path.abspath(os.curdir))[1]
 
     # Meaningful test dir from build info.
     # TODO: Organize results by day test was run. Could replace with git-hash
     whoami = getpass.getuser() or 'nobody'
-    ttime = time_now.strftime("%H_%M_%S")
-    user_tag = 'NOT_CI_{}_{}'.format(whoami, ttime)
+    user_tag = 'NOT_CI_{}'.format(whoami)
     build_tag = os.environ.get('BUILD_TAG', user_tag)
-    build_suffix = os.environ.get('BUILD_MATRIX_SUFFIX', 'standalone')
-    testdir = "{}_{}_{}".format(testname, build_tag, build_suffix)
-
-    dt = time_now.strftime("%d%b%YT")
-    tree = os.path.join(results_root, dt, testdir) + os.sep
-    schema_pattern = ['*.log']
+    date = TIME_NOW.strftime("%Y-%m-%d")
+    build_matrix_suffix = os.environ.get('BUILD_MATRIX_SUFFIX', '0')
+    subdir = '{}_{}_{}'.format(date, build_tag, build_matrix_suffix)
+    tree = os.path.join(results_root, subdir, testname) + os.sep
+    schema_pattern = []
+    # Upload all log files
+    schema_pattern.append('*.log')
```

Comment:
Why expand `schema_pattern = ['*.log']` into two different lines?

Reply:
To make it clear what is happening.  We're creating a list for the patterns.  And we're also pre-loading all the `*.log` files in there whether you want them or not.  😄

Reply:
For `jwst`, there are no output log files, so this is non-functional for us.  Perhaps this should be switchable?

Reply:
Making `schema_pattern` a keyword sounds good to me.

Reply:
Anyway, we can think about that as a separate PR...